src/stepper/index.tsx

```diff
@@ -68,7 +78,10 @@ export default class Stepper extends PureComponent <StepperProps, StepperState>
     this.props.onChange({ name: this.props.name, value })
   }
 
-  handleMouseDown = (fn: () => void) => () => {
+  handleButtonDown = (fn: () => void, isTouchEvent:Boolean = false) => () => {
```

Comment:
no need to precise `:Boolean` since you put an explicit default value

Reply:
And no need to pass this variable as an argument since it's gonna be available everywhere

Reply:
Ok for `:Boolean` but I'll keep this argument cause I'm using it depending on callback event. In case of touch screen some mouse event are triggered, so this argument allow me to cancel mouse event on mobile for example.

Reply:
We finally use touchEvent check an other way to apply appropriate Button events.